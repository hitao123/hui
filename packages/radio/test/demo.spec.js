describe('Radio component', () => {
  it('should show Radio', () => {

  });
});
