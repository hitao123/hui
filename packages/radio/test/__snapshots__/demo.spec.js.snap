// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div>
    <div class="h-radio-group">
      <div class="h-radio"><span class="h-radio__input"><input type="radio" value="1" checked="checked" class="h-radio__control"> <i class="h-icon h-icon-checked" style="color:undefined;font-size:undefined;"></i></span> <span class="h-radio__label">单选一</span></div>
      <div class="h-radio"><span class="h-radio__input"><input type="radio" value="2" class="h-radio__control"> <i class="h-icon h-icon-check" style="color:undefined;font-size:undefined;"></i></span> <span class="h-radio__label">单选二</span></div>
    </div>
  </div>
  <div>
    <div class="h-radio-group">
      <div class="h-radio h-radio--disabled"><span class="h-radio__input"><input type="radio" disabled="disabled" value="1" class="h-radio__control"> <i class="h-icon h-icon-check" style="color:undefined;font-size:undefined;"></i></span> <span class="h-radio__label">单选一</span></div>
      <div class="h-radio h-radio--disabled"><span class="h-radio__input"><input type="radio" disabled="disabled" value="2" checked="checked" class="h-radio__control"> <i class="h-icon h-icon-checked" style="color:undefined;font-size:undefined;"></i></span> <span class="h-radio__label">单选二</span></div>
    </div>
  </div>
</div>
`;
